packages/api/src/hooks/useRemoteConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+import initData from '../features.json';
+
+const remoteConfigQuery = async function () {
+    if (!process.env.REMOTE_CONFIG_URL) {
+        throw new Error('Remote Config URL is not set!');
+    }
+    const response = await fetch(process.env.REMOTE_CONFIG_URL);
+    if (!response.ok) {
+        throw new Error('Remote Config Server is out of reach!');
+    }
+    return response.json();
+};
+
+function useRemoteConfig() {
+    return useQuery({
+        queryKey: ['remoteConfig'],
+        queryFn: remoteConfigQuery,
+        initialData: initData,
```

Comment:
this is the fallback option in case that Remote Config server is out of reach for any reason.

Reply:
Usually it is called `default`